packages/api/src/controllers/taskController.js

```diff
@@ -39,6 +39,51 @@ const adminRoles = [1, 2, 5];
 //   return false;
 // };
 
+async function getTaskAssigneeAndFinalWage(farm_id, user_id, task_id) {
```

Comment:
I wonder if these two functions would actually be better suited in the model rather than the controller, since models are usually where data manipulation functions live. @kathyavini do you have any thoughts?